packages/onr-common/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "onr-common",
```

Comment:
Should we let the name start with scope `onr/` ?

Reply:
i was just trying to avoid '/' in module name in fear of confusion.